### ISSUE TITLE
unusedArguments adds an underscore to closure argument types

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#229]
This
```swift
var handler = { (_ sender: Button, index: Int, color: UIColor) -> Void in }
```

gets turned to this:

```swift
var handler = { (_: Button, _: _Int, _: _UIColor) -> Void in }
```

The following (what it probably should have been) is not modified.

```swift
var handler = { (_ sender: Button, _ index: Int, _ color: UIColor) -> Void in }
```

**Comment by nicklockwood** [comment]
@ChristopherRogers this is deliberate - since those arguments are unused they don’t need to be named.

**Comment by yonaskolb** [comment]
@nicklockwood But the types are messed up, eg `_Int`

**Comment by nicklockwood** [comment]
@ChristopherRogers @yonaskolb oh sorry, I missed that. OK, yes that's a bug. I'll fix it.

**Comment by nicklockwood** [comment]
@ChristopherRogers fixed in 0.33.4
